gradle-baseline-java/src/main/groovy/com/palantir/baseline/extensions/BaselineErrorProneExtension.java

```diff
@@ -57,7 +57,8 @@
             "ArrayEquals",
             "MissingOverride",
             "UnnecessaryParentheses",
-            "PreferJavaTimeOverload");
+            "PreferJavaTimeOverload",
+            "UnnecessaryLambda");
```

Comment:
I recall this creating code that didn’t compile, might want to test it on something large and internal (or atlas) first

Reply:
@dansanduleac you can use the hot new `./gradlew compileJava compileTestJava -PerrorProneApply=UnnecessaryLambda`

Reply:
I've tested it on a project where this was causing trouble and it produced no fixes... if you recall this way, then maybe i can just remove it from here and let people deal with it